.github/workflows/main.yml

```diff
@@ -23,8 +23,10 @@ jobs:
     - name: Generate report
       run: bash ./run.sh
     - name: Deploy GitHub pages
```

Comment:
Can I ask whether the original personal access token will work after making this change?

Reply:
No, we don't want to use the personal access token in GitHub Actions as the GitHub token is sufficient.